src/index.html

```diff
@@ -11,8 +11,52 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute in the link tag should point to a `.css` file, not `.scss`. Browsers cannot directly interpret SCSS files; they need to be compiled into CSS first.